config/authentication/login.cfg

```diff
@@ -1,16 +1,16 @@
-radmin:scNF1tajgmyILrZgSlY/0g\=\= Sc3qfygFoBq3IWxubfRVIWlj+6DpdVJMRHBbom/8F1UcdYjO5nwDstoYhG4qChk2iCG9xyrFaFt4RfSfjUbpxyFSBcGbYPhSSgwyrneGx4bdrTxgDFjjfnWVeZsvjYkdbFj60ry+U4mCr1Oz0sNwdQ8LUUIaB24DRM1ttHb/UsY\=
-rm:RZGx1vYQYoG280nslk9lbw\=\= ZdXareXM9uxs5fMc2FsqrQPaiHZW8AHcojeeJpuwstIzos5HX0T6fHkvRB2+cRd7IwydDP9M2ofoJ3Ut1VrkJAXBC4hgIqo0C5Dfu2LPl51cvaxhb649ryuP0+apTpgju3YYuXRo945Zq6XO5WcZn/mCP/4hs5IbG+89myXboNA\=
-scheduler:ESsKwjnqdXvuNw02PC7yQQ\=\= MVglgGC3jTMKqvc6jbTDnxkV/7eRbTNzXqSSNsLEdsvKSi/qvfzQsIJXpBJ0alpievWrmuGq07a20jET7I7iMF3HzVN3spfNvY7nN9eVxzhf69czHeF/aUxiZwgaImNzff7ygnGLwmN1+LRL8ab7jsf2kNmvd7RCU/k9/53ObdQ\=
-subadmin:bim9owbtzHsm46w6CcWRJg\=\= Q+QD/wppzsl2vJnDqtQZnuHbEINkKlyIXea175V4h6scHGuiscjdytaBWH/wKUbm+l1Hz3a3FbwZztSMYBXRqemvFgv+4+IgZ1moYrTBdfBnj+zjwOicYLS7Wl4MW5KJ6l8RxyfoIwLzTQsYLYgKzRkmIRE/YC7End3fZDLOhj8\=
-test:IcjMbCeMvS5q187XLw7pfw\=\= JGhYLOcB9P5l9AMJzZOhZd4O8h9RT+VfWRF1YMr+zRoduQiqBqReKVGb6OTq+lkQPJukSLsUcr7mznOFxAADSIdaQyzaxU9og1y34SKr+4qCihizq1ckHcqSbFmV1Rn2WuAJE+LRKMyUHfV2Q7pqYpClsCugm+TsV7vLSSVjCGk\=
-test_executor:s/MsIjSxmEZP2Sygjaw/nw\=\= qEEOlvzx3dBSrxYcre3hRcq1KThwSvPPbQti+KnaMHsfHfSgb69eFIiunaAvSEPOLRjqo64GVsPLbMztUuQFoqhddQ4bbwfZCWbxnzhv25sOc35SsytdxxxCYZh94ZnlnIXNb9GnZz8xYyQ/HC4HeB2hm6aJExegN7gXHBVhx68\=
-user:NOr/v7CayeQW60Z/3W/mtw\=\= WLsK6Z6sF0P1+EfdnsvGqrkpLAQIuMzlmE93Ehn3hGhAxb5HYGtWaT5ZSKzJtqkkepHGGPP67+QVPlqS+yYr2iffihGVT2qc7tMwQJLOqyKhTL0ALnlqCHMRAmXzBSTN0lzf6vfx8Au/fROYEECRxTeRV3x2CjZUV3prG4SIp9k\=
-watcher:Nxd2xNqf2kVieip2RFR8Jw\=\= V3fa83IdzbAipfOu5iH6YhG1njgInISsb2ZuNzyVFCkeVgxEgvVO8X5CtwdyIijsUMuFLaMGdRBVbu5qfyALLCG5WhN1nHtGeXFl2bRQW3uiYW4fqlAOgLrJVC7sZJbagy8GtTMBWGxSKv5IpXTO7d5lCkj3JXdk+C6iD+sIad4\=
+admin:$2a$12$2IVFh1cV6/m8yUuYF4e/tO7jyGHK3R7L5dVb/.84ibvWh1i/16n8K
+citizen-ds:$2a$12$fTWPz2nF6mwZaVQBHRCB0OWHWS9R/1mCq/QMji3keuoSuVMOQs/Be
+demo:$2a$12$CdZ24h4ux4Jx/qkBkgy/LeBwmxQknwj7rO9LHwMPN467IBv1r/LYq
+expert-ds:$2a$12$sIW7Kd9CoQKLMuYB1B78BuBixTdZxGIdAqsNCq3kn/xZdNYMDaWTa
+guest:$2a$12$WUUolhvUN5/vrLTUuQqipeH3SQtGzAmNZ3P2jA62B5BxCVSJD9peq
+nsadmin:$2a$12$3FByPmqKMX0BuVSA8QpvKuXnY/IFFhc8Zz.Sk/4Fk6CrHHjyUxFAy
+nsadmin2:$2a$12$ml3UHl8QeUxgjMFcAGTB8OGDvl4tGYZA7LBBKgEEyQ4x7Qp0lMwTS
+provider:$2a$12$a94e.v2qzMPKp7nn/CMMFu3G28WdvWSmWs98fPbVJeiJPpm6WCfc.
+radmin:$2a$12$/uMxFHAPidOcmjemfmv/DOLEUTx5L3cJKV/wKS76yeKugzB21t106
+rm:$2a$12$reLZNyffRQmZ1ACZNn85x.rosnt4AXEnsn/tfEq6bxw0mCT2Qwbl6
+scheduler:$2a$12$.cmyzTeCiFo8pxr6k9mCF.rNqwUMS2OY2zwlRh1lTQbjkqePd9O5q
+subadmin:$2a$12$/V4AQYyeR2U9EyCQ8hq52.BN/Xul1vSzIy0FuGil6/xsuoRjufUw2
+test:$2a$12$BsCEuxiiGFiPXh2Bu3EzfeVlV09O2nIfXhq28HuLOUmruRfbhUUZ.
+test_executor:$2a$12$LmF/0h1e8AEWV6YQkLqNxOZ4IvquPOR5Nf/yw2/KuEJaPS2bSo1..
+user:$2a$12$GWJnc6LaQNNHoCtusN626efZiCEWCA5rBPA7jBeLiWgERmxOlTs8C
+watcher:$2a$12$ko6Hf7SX.dqTY.zYKFUBLOrM1Kx8Ujuvb8uGAVKUTgvRmzZYCXdUG
```

Comment:
Many users are not uptodate with the doc like test users:
https://trydev2.activeeon.com/doc/admin/ProActiveAdminGuide.html#_groups

Reply:
it's the doc that's not uptodate :-)